.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
is this going to create any issue if someone does not have nix installed, or have nix but wants to use stack?

Reply:
Hm, I think only if they have `direnv` installed.

Reply:
just because I'm ignorant on this, what's the benefit of having this file?

Reply:
Actually, they'll need to use `direnv allow` for direnv to have any effects, but I'll mention something about it in the README just in case.